mrtt-ui/src/components/RestorationAimsForm/RestorationAimsForm.js

```diff
@@ -0,0 +1,66 @@
+import { FormLabel } from '@mui/material'
+import { useState } from 'react'
+import { useForm } from 'react-hook-form'
+import { FormQuestionDiv, MainFormDiv, SectionFormTitle } from '../../styles/forms'
+import ButtonSubmit from '../ButtonSubmit'
+import CheckboxGroupMangroveWithLabel from '../CheckboxGroupMangroveWithLabel'
+import restorationAimsQuestionsAndAnswers from './restorationAimsQuestionsAndAnswers'
+
+const RestorationAimsForm = () => {
+  const [isSubmitting, setIsSubmitting] = useState(false)
+  const reactHookFormInstance = useForm()
+
+  const { handleSubmit: validateInputs } = reactHookFormInstance
+
+  const handleSubmit = () => {
+    setIsSubmitting(true)
+    // hit api here
+    setIsSubmitting(false)
+  }
+  const generateOptionsObjectFromAnswers = (answers) =>
+    answers.map((answer) => ({
+      label: answer,
+      value: answer
+    }))
+  const ecologicalAimsOptions = generateOptionsObjectFromAnswers(
+    restorationAimsQuestionsAndAnswers[1].answers
+  )
+  const socioEconomicAimsOptions = generateOptionsObjectFromAnswers(
+    restorationAimsQuestionsAndAnswers[2].answers
+  )
+  const otherAimsOptions = generateOptionsObjectFromAnswers(
+    restorationAimsQuestionsAndAnswers[3].answers
+  )
+  return (
+    <MainFormDiv>
+      <SectionFormTitle>Restoration Aims</SectionFormTitle>
+      <form onSubmit={validateInputs(handleSubmit)}>
+        <CheckboxGroupMangroveWithLabel
```

Comment:
love how clean this looks, but still not 100% clear on the code - maybe we can screenshare to go over it!